README.md

```diff
@@ -18,7 +18,7 @@
 [cov-url]: https://codecov.io/gh/LCSB-BioCore/COBREXA.jl
 
 [contrib-img]: https://img.shields.io/badge/contributions-start%20here-green
-[contrib-url]: https://github.com/LCSB-BioCore/COBREXA.jl/blob/master/.github/CONTRIBUTING.md
+[contrib-url]: https://lcsb-biocore.github.io/COBREXA.jl/dev/howToContribute/
```

Comment:
The URL should be `stable` here I think

Reply:
contribution kindof implies development so I pointed it to the (likely more relevant) dev version

Reply:
contributing guide should be the one from `master`, i.e. the stable version. 

Although a developer might consult the development guide, it should be the stable version

Reply:
ok